Extension/src/packageManager.ts

```diff
@@ -183,11 +183,10 @@ export class PackageManager {
             } catch (error) {
                 retryCount += 1;
                 lastError = error;
-                if (retryCount >= MAX_RETRIES) {
+                if (retryCount > MAX_RETRIES) {
```

Comment:
we used to do 5 retries, but currently are only doing 4 because of retryCount + 1 before this check